config/build.sh

```diff
@@ -12,8 +12,8 @@ set -e -x
 VERSION="$(date -u +%Y-%m-%d)-$(git rev-parse --short HEAD)"
 
 cd "${BASEDIR}"
-docker build -f cmds/http-gateway/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-docker build -f cmds/grpc-backend/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
+docker build -f cmds/http-gateway/Dockerfile  . -t $DOCKER_URL/http-gateway:$VERSION
+docker build -f cmds/grpc-backend/Dockerfile  . -t $DOCKER_URL/grpc-backend:$VERSION
 
-docker push gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-docker push gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
+docker push $DOCKER_URL/http-gateway:$VERSION
```

Comment:
Let's make sure to exercise build.sh (and the other scripts) to ensure they still work after the changes.  We probably didn't do that as of last revision as the HTTP gateway wouldn't have deployed correctly without the suffixes.

Reply:
done!